colorcoder/ColorPairTest.java

```diff
@@ -0,0 +1,30 @@
+package colorcoder;
+
+public class ColorPairTest 

```

Comment:
When class name is ColorPairTest then it should hold the responsibilities towards test ....then why these functionalities "GetColorFromPairNumber" and "GetPairNumberFromColor" encapsulated ?